src/core.c/Stash.pm6

```diff
@@ -50,7 +51,7 @@ my class Stash { # declared in BOOTSTRAP
                             nqp::getattr(self, Hash, '$!descriptor')),
                         assignval)
                 );
-                nqp::bindattr(self, Map, '$!storage', $storage);
+                nqp::atomicbindattr(self, Map, '$!storage', $storage);
```

Comment:
Wouldn't this fail on the JVM backend ?

Reply:
No idea. I picked the op from BOOTSTRAP where it's used in `atomic_store`. The op is not `#?if`-wrapped.